build.sbt

```diff
@@ -125,7 +125,7 @@ val testDependencies = Seq(
   "us.codecraft" % "xsoup" % "0.3.1" % "test,it",
   "org.mockito" % "mockito-core" % "3.0.0" % "test,it",
   "org.pegdown" % "pegdown" % "1.6.0",
-  "uk.gov.hmrc" %% "reactivemongo-test" % "5.0.0-play-28" % "test, it"
+  "uk.gov.hmrc.mongo" %% "hmrc-mongo-test-play-27" % "0.56.0" % "test, it"
```

Comment:
Why don't we use the play-28 version?

Reply:
The migration guide doesn't stipulate upgrading the test library version.  We have checked other projects that have done the migration and they still use version 2.7 for testing... We tried updating the test framework to 2.8 and it broke several tests. Perhaps the test library version does not need upgrading? Suggest tackling this as a separate technical task. Will also ask the platOps team.